# HG changeset patch
# Parent  f3777e19eb31224131d0d8ca647c54b3521171b9
lightsd: Update LIFX products ids (closes GH-23)

diff --git a/lifx/bulb.c b/lifx/bulb.c
--- a/lifx/bulb.c
+++ b/lifx/bulb.c
@@ -66,8 +66,20 @@
         return "White 800";
     case 0x12:
         return "White 900 BR30";
+    case 0x14:
+        return "Color 1000 BR30";
     case 0x16:
         return "Color 1000";
+    case 0x1b:
+        return "A19";
+    case 0x1c:
+        return "BR30";
+    case 0x1d:
+        return "A19+";
+    case 0x1e:
+        return "BR30+";
+    case 0x1f:
+        return "Z";
     default:
         return "Unknown";
     }
